File: Android_Java/HttpRequest/app/src/main/java/com/example/sergio/httprequest/MainActivity.java
package com.example.sergio.httprequest;

import android.support.v7.app.AppCompatActivity;
import android.os.Bundle;
import android.widget.Toast;

import org.json.JSONException;
import org.json.JSONObject;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        String json = "{\n" +
                            "cod: 1,\n" +
                            "message: 'London'" +
                      "}";

        City city = null;

        //Parsear el json
        try {
            //Recuperar los datos del json
            JSONObject mJson = new JSONObject(json);
            int id = mJson.getInt("id");
            String name = mJson.getString("name");

            //Crear la ciudad
            city = new City(id, name);

        } catch (JSONException e) {
            e.printStackTrace();
        }

        Toast.makeText(this, city.getId() + " -- " + city.getName(), Toast.LENGTH_LONG).show();
    }
}

File: Android_Kotlin/MyLibrary/app/src/main/java/com/example/sergio/mylibrary/MainActivity.kt
package com.example.sergio.mylibrary

import android.os.Bundle
import android.support.v7.widget.Toolbar
import com.example.mylibrary.ToolbarActivity
import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : ToolbarActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        toolbarToLoad(toolbar as Toolbar)
    }
}
